Checkout/Checkout/Source/Validation/Validators/CardNumberValidator.swift

```diff
@@ -44,7 +44,8 @@ final class CardNumberValidator: CardNumberValidating {
   }
     
   func validateCompleteness(cardNumber: String) -> Result<ValidationScheme, ValidationError.CardNumber> {
-    let cardNumber = cardNumber.filter { !$0.isWhitespace }
+      "".first?.isWhitespace
```

Comment:
please add more details for this line

Reply:
That was a big omission from my side when I filtered the commit 🤦‍♂️. When I'm not sure about syntax I usually just write it in the nearest open Xcode but I'd normally see it and not commit.

Thanks for noticing it.